code/_helpers/text.dm

```diff
@@ -433,6 +433,7 @@ proc/TextPreview(var/string,var/len=40)
 	text = replacetext(text, "\[fontblue\]", "<font color=\"blue\">")//</font> to pass travis html tag integrity check
 	text = replacetext(text, "\[fontgreen\]", "<font color=\"green\">")
 	text = replacetext(text, "\[/font\]", "</font>")
+	text = replacetext(text, "\[redacted\]", "<span class=\"redacted\">R E D A C T E D</span>")
```

Comment:
Could you add this to `/datum/codex_entry/pen` too? I think it will be forgotten if not done in this PR.

Reply:
Nvm, as you pointed out this is digital pencode only.